### DISCUSSION TITLE
any plan to support react (next) with Mantine?

### DISCUSSION THREAD

**Discussion by tjiahen** [onlook-dev/onlook#35] [Questions] [answered]
any plan to support react (next) with Mantine?

**Comment by Kitenite** [comment, marked as answer]
Hey @wizztjh ,
No explicit plan for mantine atm. However, you can see here where the classes are being generated from CSS and written to the React nodes. For Mantine I would generate the mantine styles from CSS and then write it into the style node instead of className node.

Generate Tailwind from CSS: https://github.com/onlook-dev/studio/blob/main/app/src/lib/editor/engine/code/index.ts#L40
Write tailwind classes to React: https://github.com/onlook-dev/studio/blob/main/app/electron/main/code/index.ts#L40

I will keep checking this and prioritize higher if there's more demand :)
